### PR TITLE
[ISSUE #332]Add Jdbc Source Connector and Do Unit Test

### PR DESCRIPTION
**Pull request by yuchenlichuck**
## What is the purpose of the change

Add Jdbc Source Connector and Do Unit Test

## Brief changelog

1. create pom.xml, Config.java, JdbcSourceConnector.java, JdbcSourceConnectorTest.java

## Verifying this change

I have done the unit test for JdbcSourceConnector.

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [X] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [X] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [X] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [X] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [X] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/connector/JdbcSourceConnector.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class JdbcSourceConnector extends SourceConnector {
+    private static final Logger log = LoggerFactory.getLogger(JdbcSourceConnector.class);
+    private KeyValue config;
+
+    @Override
+    public String verifyAndSetConfig(KeyValue config) {
+        System.out.println(11+config.toString());
```
Comment (duhenglucky) [inline rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/connector/JdbcSourceConnector.java]: Why used System.out.println in here and what's the meaning of this print?

File: rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/connector/JdbcSourceConnector.java [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        log.info("JdbcSourceConnector verifyAndSetConfig enter");
+        for (String requestKey : Config.REQUEST_CONFIG) {
+            System.out.println(12+requestKey);
+            if (!config.containsKey(requestKey)) {
+                return "Request config key: " + requestKey;
+            }
+        }
+        this.config = config;
+        System.out.println(11+config.toString());
```
Comment (duhenglucky) [inline rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/connector/JdbcSourceConnector.java]: same as the last comment

File: rocketmq-connect-jdbc/pom.xml [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            <artifactId>rocketmq-openmessaging</artifactId>
+            <version>4.3.2</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>1.2</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.jms</groupId>
```
Comment (duhenglucky) [inline rocketmq-connect-jdbc/pom.xml]: Why would you depend on this library ？

File: rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/Config.java [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    public static final Set<String> REQUEST_CONFIG = new HashSet<String>() {
+        {
+            add("jdbcAddr");
+            add("jdbcPort");
+            add("jdbcUsername");
+            add("jdbcPassword");
+        }
+    };
+
+//    public String destinationType;
```
Comment (duhenglucky) [inline rocketmq-connect-jdbc/src/main/java/org/apache/rocketmq/connect/jdbc/Config.java]: It seems that this file copied from jms connector, and please remove these Irrelevant code

**Comment by yuchenlichuck** [comment]
#332

**Pull request merged**